.NET/Microsoft.Recognizers.Text.DateTime/Chinese/Parsers/DatePeriodParserChs.cs

```diff
@@ -1131,6 +1136,108 @@ private DateTimeResolutionResult ParseQuarter(string text, DateObject referenceD
             return ret;
         }
 
+        private DateTimeResolutionResult ParseDecade(string text, DateObject referenceDate)
+        {
+            var ret = new DateTimeResolutionResult();
+            int century = referenceDate.Year / 100;

```

Comment:
2001 is 21st century, not 20th. Do you account for this later? If so, this var name should be modified.